docs/tutorials/04_torch_qgan.ipynb

```diff
@@ -811,40 +634,32 @@
     }
    },
    "source": [
-    "### Results: cumulative distribution functions\n",
-    "In the final section we compare the cumulative distribution function (CDF) of the trained distribution to the CDF of the target distribution.\n",
+    "## 6. Results: Cumulative Density Functions\n",
+    "In this section we compare the cumulative distribution function (CDF) of the trained distribution to the CDF of the target distribution.\n",
     "\n",
-    "We create a new generator for sampling that takes more shots and, hence, gives more information. Recall, the sampling quantum instance was created with a larger number of shots. Then, we set the weights of the sampling generator to the values obtained in the training process."
+    "First, we generate a new probability distribution disabling gradient evaluation."
```

Comment:
Will the audience understand why its done with `disabling of gradient evaluation.` Its been called out here in the text explicitly - since it got called out I guess that would be a question I have.

Reply:
It is a common pattern working with pytorch, when a model is trained and there's no need anymore to evaluate gradients, you either switch the model into the evaluation mode by calling`model.eval()` or call `no_grad()` to turn off autograd.
Slightly rephrased.